lib/linguist/languages.yml

```diff
@@ -3369,6 +3369,8 @@ Perl:
   - ".psgi"
   - ".t"
   filenames:
+  - Makefile.PL
```

Comment:
This is redundant, because it'll be matched against the `*.pl` file extension.

Reply:
@Alhadis That extension is shared between Perl, Perl 6, and Prolog. Adding the `Makefile.PL` might improve detection for that particular case. Are we sure `Makefile.PL` is always Perl and not Perl 6 though?

Reply:
I stand corrected, then. :wink: @pchaigno knows more about the Bayesian classifier than I do.

I don't recall ever seeing a Perl 6 version of `Makefile.PL`... it's always ever been used by Perl 5 modules downloaded/installed from CPAN. I could be wrong though.

Reply:
Pretty sure. Perl6 has pretty much normalized to .pl6 or .p6 now. It's not using this particular file, either. This is the repo for all modules https://github.com/moritz/perl6-all-modules/search?utf8=%E2%9C%93&q=%22Makefile.PL%22&type= and it's not there even once. Makefile.PL is kind of a Rakefile used for some CPAN distributions; Perl6 uses zef and META6.json

Reply:
> @pchaigno knows more about the Bayesian classifier than I do.

I don't know much about the classifier either, but in this particular case, there's a [specific filename strategy](https://github.com/github/linguist/blob/dd3b1eec911072ca051ecccabc03a47bafa2c76c/lib/linguist.rb#L62) before we reach the classifier.

PS: Not sure why I keep using "particular case" today...

Reply:
Sounds good to me, then. :+1: The bulk of my familiarity with `Makefile.PL` is limited to seeing those files whoosh past the terminal when installing CPAN modules... :grinning:

Reply:
Is there any other change you want me to do here?

Reply:
Ping... Is there anything missing?

Reply:
No, it looks fine to me. For the sake of quicker review, it might be better to omit `.pod6` since #3366 is more appropriate for discussion its inclusion.

Reply:
Eliminated `.pod6`. I can try and work on that one, although I can't add or modify that pull request...

Reply:
Don't worry about #3366. :) I believe that PR needs formal approval from GitHub staff more than anything.